### PR TITLE
Do not ignore *.meta files

### PR DESCRIPTION
**Pull request by richkzad**
Unity docs make it clear that meta files should be included in your repo: https://docs.unity3d.com/Manual/ExternalVersionControlSystemSupport.html

### CODE REVIEW

**Comment by van800** [comment]
Hi! 
What is the scenario in which meta files are useful for you?

**Comment by richkzad** [comment]
This blog post seems to explain it pretty well:
https://blog.forrestthewoods.com/managing-meta-files-in-unity-713166ee3d30

From what I understand, assets in Unity are referenced in projects based on their guid, which is stored in the .meta files. If you don't include the .meta files, another person working on the project may end up creating their own .meta files for assets, and then the project will be referencing different guids, causing missing references.

tl;dr: not including .meta files can cause the project references to stop working correctly

**Comment by van800** [comment]
For some reason similar plugins were not including meta.
https://github.com/dotBunny/VSCode/tree/master/Plugins/Editor

**Comment by richkzad** [comment]
Oh, you may have guessed it -- this is my bad.

This may sound strange, but I forgot that this was the gitignore for this repo.

I've been looking at the github/gitignore repo recently, and I assumed this gitignore was a model gitignore for using Unity3dRider in your project.

Whoops! Sorry for the distraction.
